packages/sdks/src/components/content/content.lite.tsx

```diff
@@ -56,11 +56,7 @@ export default function ContentComponent(props: ContentProps) {
 
     registeredComponents: [
       ...getDefaultRegisteredComponents(),
-      ...(props.customComponents?.filter(({ models }) => {
-        if (!models?.length) return true;
-        if (!props.model) return true;
-        return models.includes(props.model);
-      }) || []),
```

Comment:
I see that you permanently got rid of this filtering for `registeredComponents`.

For the sake of consistency, let's keep this logic and move it to:
 https://github.com/BuilderIO/builder/blob/25f00aca07a395df8a5b663ffc963d0e8c808570/packages/sdks/src/components/block/block.lite.tsx#L118-L123

To make sure that the registered components and the rendered components perfectly match.

Reply:
sure @samijaber makes sense, let me do that

Reply:
@samijaber added the restriction part, do you think we should also filter out registered components that we expose to the users when they do `shouldReceiveBuilderProps: { builderComponents: true }` here?
https://github.com/BuilderIO/builder/blob/8a209a99793802ae09aeec0664455caaadb7fded/packages/sdks/src/components/block/block.lite.tsx#L191-L194

Reply:
Ah yes, good catch. Let's do that for consistency as well

Reply:
done!